user/incus/template.py

```diff
@@ -0,0 +1,81 @@
+pkgname = "incus"
+pkgver = "6.4.0"
+pkgrel = 10
+build_style = "go"
+make_build_args = ["./..."]
+hostmakedepends = [
+    "automake",
+    "libtool",
+    "bash",
+    "git",
+    "go",
+    "go-md2man",
+    "less",
+    "pkgconf",
+]
+makedepends = [
+    "acl-devel",
+    "cowsql-devel",
+    "libseccomp-devel",
+    "libcap-devel",
+    "linux-headers",
+    "libuv-devel",
+    "lxc-devel",
+    "raft-devel",
+    "sqlite-devel",
+    "udev-devel"
+]
+depends = [
+    "libvirt",
+    "acl",
+    "acl-progs",
+    "lxc",
+    "rsync",
+    "squashfs-tools",
+    "xz",
+    "dnsmasq",
+    "iptables",
+    "attr-progs",
+    "util-linux"
+]
+go_build_tags=[ "libsqlite3" ]
+pkgdesc = "Powerful system container and virtual machine manager"
+maintainer = "tj <tjheeta@gmail.com>"
+license = "Apache-2.0"
+url = "https://github.com/lxc/incus"
+source = f"{url}/archive/v{pkgver}.tar.gz"
+sha256 = "c6ad8260a4954b0e4f77de66220d5450290b57858f8c16ee83f12996eea86db4"
+# void ignores TestConvertNetworkConfig (https://github.com/void-linux/void-packages/blob/master/srcpkgs/incus/template)
```

Comment:
linking to unstable urls that break in a week is not useful, either use a permalink or actually pass this in make_check_args and enable the test

Reply:
used permalink for now